action.yaml

```diff
@@ -0,0 +1,196 @@
+            -v "${{ steps.workspace.outputs.constructor-workspace }}:${{ steps.workspace.outputs.constructor-workspace }}"
+            ${{ inputs.standalone-location && format('-v $(dirname {0}):$(dirname {0})', inputs.standalone-location) || '' }}
+            ${{ inputs.container-arch && format('--platform {0}', inputs.container-arch) || '' }}
+            -e GITHUB_WORKSPACE
+            ${{ inputs.container-image }}
+            /bin/bash -eo pipefail {0}
+        )
+        echo "container-command=${DOCKER_CMD[@]}" >> ${GITHUB_OUTPUT}
+      shell: bash
+
+    - name: Build installer
+      id: build
+      env:
+        AZURE_SIGNTOOL_KEY_VAULT_CLIENT_ID: ${{ inputs.azure-signtool-key-vault-client-id }}
+        AZURE_SIGNTOOL_KEY_VAULT_CERTIFICATE: ${{ inputs.azure-signtool-key-vault-certificate }}
+        AZURE_SIGNTOOL_KEY_VAULT_SECRET: ${{ inputs.azure-signtool-key-vault-secret }}
+        AZURE_SIGNTOOL_KEY_VAULT_TENANT_ID: ${{ inputs.azure-signtool-key-vault-tenant-id }}
+        AZURE_SIGNTOOL_KEY_VAULT_URL: ${{ inputs.azure-signtool-key-vault-url }}
+        CONSTRUCTOR_PFX_CERTIFICATE_PASSWORD: ${{ inputs.constructor-pfx-certificate_password }}
+      run: |
+        echo "::group::Create environment"
+        # Define variables here instead of in an enviornment to simplify Docker command
+        CONSTRUCTOR_WORKSPACE="${{ steps.workspace.outputs.constructor-workspace }}"
+        OUTPUTS_FILE="${{ steps.workspace.outputs.outputs-file }}"
+        ENVIRONMENT_YAML="${{ inputs.environment-yaml-file || steps.environment-str.outputs.environment-yaml-file }}"
+        CONDA_ROOT="${{ inputs.conda-root }}"
+
+        UNAME=$(uname)
+        if [[ "${UNAME}" == MINGW* ]] && [[ -n "${CONDA_ROOT}" ]]; then
+          CONDA_ROOT=$(cygpath "${CONDA}")
+        fi
+        RECIPE_DIR="${GITHUB_WORKSPACE}/${{ inputs.recipe-directory }}"
+        STANDALONE_EXE="${{ inputs.standalone-location }}"
+
+        if [[ -d "${CONDA_ROOT}" ]]; then
+            . "${CONDA_ROOT}/etc/profile.d/conda.sh" && conda activate
+        fi
+        if [[ -n "$(command -v conda)" ]]; then
+            CONDA_BIN=conda
+        elif [[ -n "$(command -v mamba)" ]]; then
+            CONDA_BIN=mamba
+        elif [[ -n "${STANDALONE_EXE}" ]]; then
+            CONDA_BIN="${STANDALONE_EXE}"
+        else
+            echo "::error::Could not find conda or mamba binary."
+            exit 1
+        fi
+
+        PREFIX="${CONSTRUCTOR_WORKSPACE}/constructor"
+        ${CONDA_BIN} env create -p "${PREFIX}" --file "${ENVIRONMENT_YAML}" -y
+        echo "::endgroup::"
+
+        echo "::group::Construct the installer"
+        ARTIFACTS_DIRECTORY="${CONSTRUCTOR_WORKSPACE}/build"
+        mkdir -p "${ARTIFACTS_DIRECTORY}"
+        EXTRA_CONSTRUCTOR_ARGS=""
+        if [[ -n "${STANDALONE_EXE}" ]]; then
+            EXTRA_CONSTRUCTOR_ARGS+=" --conda-exe ${STANDALONE_EXE}"
+        fi
+        ${CONDA_BIN} run -p "${PREFIX}" \
```

Comment:
Pass the flag here to _not_ capture output and instead live stream stdout/stderr. Not sure if it's the same for both conda & micromamba.